test/test.sql

```diff
@@ -1,7 +1,7 @@
 DROP TABLE person_test;
```

Comment:
In general, to avoid seeing ORA-942 when the table doesn't exist, you could do something like:
    BEGIN
      EXECUTE IMMEDIATE 'DROP TABLE xyz';
      EXCEPTION
       WHEN OTHERS then
       if SQLCODE <> -942 then -- ignore "does not exist"
      raise;
       end if;
    END;
    /